yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/util/Decryption.java

```diff
@@ -0,0 +1,29 @@
+package com.yoti.api.client.spi.remote.util;
+
+import static javax.crypto.Cipher.DECRYPT_MODE;
+
+import static com.yoti.api.client.spi.remote.call.YotiConstants.ASYMMETRIC_CIPHER;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.BOUNCY_CASTLE_PROVIDER;
+
+import java.security.GeneralSecurityException;
+import java.security.PrivateKey;
+
+import javax.crypto.Cipher;
+
+import com.yoti.api.client.ProfileException;
+
+public class Decryption {
```

Comment:
can we come up with a different kind of naming? `CryptoHelper` or something along those lines?

Reply:
It's now named DecryptionHelper.